File: Array/Pascal's-triangle/answer.js
var generate = function(numRows) {
    let ans = [[1]]
    if(numRows > 1) {
        for(let i = 1; i < numRows; i++) {
            if(i == 1) {
                ans.push([ans[0][0],1])
            } else {
                let temp = []
                for(let n = 0; n < ans[i - 1].length - 1; n++) {
                    let mid = ans[i - 1][n] + ans[i - 1][n+1]
                    temp.push(mid)
                }
                temp = [1,...temp,1]
                ans.push(temp)
            }
        }
    }
    return ans
};

console.log(generate(6))

File: Array/Binary-search/counting-element-in-two-arrays/answer.py

def countEleLessThanOrEqual(arr1,n1,arr2,n2):
    #returns the required output
    arr2.sort()
    ans = []
    for i in range (0, len(arr1)):
        low = 0
        high = n2 - 1
        res = 0
        while low <= high:
            mid = low + (high - low) // 2
            if arr1[i] >= arr2[mid]:
                low = mid + 1
                res = low
            else:
                high = mid - 1
        ans.append(res)
    return ans


arr1 = [1,2,3,4,7,9,0]
arr2 = [1,1,2,1,1,4]
print(countEleLessThanOrEqual(arr1,len(arr1),arr2,len(arr2)))